nilearn/decoding/space_net.py

```diff
@@ -636,12 +658,24 @@ class BaseSpaceNet(LinearModel, RegressorMixin, CacheMixin):
          Each pair are are the list of indices for the train and test
          samples for the corresponding fold.
 
-    `cv_scores_` : ndarray, shape (n_alphas, n_folds) or (n_l1_ratios, n_alphas, n_folds)
+    `cv_scores_` : ndarray, shape (n_folds, n_alphas) or (n_l1_ratios, n_folds, n_alphas)
         Scores (misclassification) for each alpha, and on each fold
 
     `screening_percentile_` : float
         Screening percentile corrected according to volume of mask,
         relative to the volume of standard brain.
+
+    `w_` : ndarray, shape (n_classes-1, n_features) #TODO: check shapes
```

Comment:
Yes @danilo is right. So, shape `w_ should be documented as
* `(1, n_features + 1)` for two class classification problems (i.e `n_classes = 2`)
* `(n_classes, n_features + 1)` for `n_classes > 2`, and
* `(n_features,)` for regression